.github/dependabot.yml

```
@@ -0,0 +1,12 @@
+version: 2
+updates:
+- package-ecosystem: maven
+  directory: "/"
+  schedule:
+    interval: weekly
+  open-pull-requests-limit: 10
+  target-branch: master
+  reviewers:
+  - <insert-maintainers-here-one-per-line>
```

needs a change of the reviewers to be requested or needs to have the reviewers section removed.
```suggestion
  - MarkEWaite
  - gounthar
  - jmMeessen
```